iOS/ACPWeatherExample/ACPWeatherExample/WeatherExtensionInternal.m

```diff
@@ -225,12 +225,12 @@ - (void) extractMidFromIdentitySharedState:(ACPExtensionEvent *) event {
     NSDictionary *identitySharedState = [self.api getSharedEventState:ACP_IDENTITY_SHARED_STATE event:event error:&error];
     
     if (!identitySharedState) {
-        NSLog(@"%@ - Identity shared state is pending, returning nil", [self name]);
+        [ACPCore log:ACPMobileLogLevelWarning tag:@"WeatherExtensionInternal" message:[NSString stringWithFormat:@"%@ - Identity shared state is pending, returning nil", [self name]]];
```

Comment:
I would say that the pending shared state is not considered a warning, as we evaluate this again when the shared state is updated. Debug might be better in this situation